build/docker/dockerfile.tmpl

```diff
@@ -65,8 +65,8 @@ RUN chown -R redis:redis /var/opt/redislabs
 COPY --from=builder /build/bin/artifacts/ /var/opt/redislabs/artifacts/
 {% endif %}
 
-{% if REDISEARCH_POWER_TO_THE_WORKERS == "1" %}
-CMD ["sh", "-c", "redis-server --loadmodule {{MODULES_DIR}}/redisearch.so ALWAYS_USE_THREADS TRUE WORKER_THREADS $(nproc) --loadmodule {{MODULES_DIR}}/rejson.so"]
+{% if REDISEARCH_MT_BUILD == "1" %}
+CMD ["sh", "-c", "redis-server --loadmodule {{MODULES_DIR}}/redisearch.so MT_MODE MT_MODE_RCP WORKER_THREADS $(nproc) --loadmodule {{MODULES_DIR}}/rejson.so"]
```

Comment:
`MT_MODE_RCP`? For the general public?

Reply:
I agree, this basically changes the previous configuration (`ALWAYS_USE_THREADS TRUE` is equivalent to `MT_MODE_RCE`)